src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -210,4 +208,32 @@ def test_containerapp_env_certificate_e2e(self, resource_group):
 
         self.cmd('containerapp env certificate list -g {} -n {}'.format(resource_group, env_name), checks=[
             JMESPathCheck('length(@)', 0),
-        ])
\ No newline at end of file
+        ])
+
+
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northeurope")
+    @live_only()  # passes live but hits CannotOverwriteExistingCassetteException when run from recording
```

Comment:
> passes live but hits CannotOverwriteExistingCassetteException when run from recording

What is the specific information of this exception? Can't you pass the test in record mode even locally?